### PR TITLE
Override stylistic rule max line length from 120 to 160

### PR DESCRIPTION
**Pull request by woahitsjc**
- Override `max line length` from 120 to 160

### CODE REVIEW

File: packages/aquila-stylelint/index.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ module.exports = {
     "rules": {
         "@stylistic/declaration-block-semicolon-newline-after": "always",
         "@stylistic/indentation": 4,
+        "@stylistic/no-empty-first-line": null,
```
Comment (hisuwh) [inline packages/aquila-stylelint/index.js:13]: Why would we want an empty first line?

**Review by hisuwh — CHANGES_REQUESTED** [review]

File: packages/aquila-stylelint/index.js (hunk 1 above)
Comment (woahitsjc) [reply packages/aquila-stylelint/index.js:13]: @hisuwh Yeah we don't really want it. It's true by default so I've set it to null to ignore.

Comment (hisuwh) [reply packages/aquila-stylelint/index.js:13]: I would say we don't want an empty first line surely?

Comment (hisuwh) [reply packages/aquila-stylelint/index.js:13]: Was this just causing loads of errors though?

Comment (woahitsjc) [reply packages/aquila-stylelint/index.js:13]: @hisuwh Yeah it was complaining that there were empty first lines.

I have added an empty first line in the test to make sure that doesn't error.

Comment (woahitsjc) [reply packages/aquila-stylelint/index.js:13]: We can enforce it though, should be simple with `--fix`

File: packages/aquila-stylelint/package.json [hunk 2]
```diff
@@ -20,12 +20,10 @@
...
     "homepage": "https://github.com/AquilaLearning/aquila-linting#readme",
     "peerDependencies": {
+        "@stylistic/stylelint-config": "2.x",
         "stylelint": "16.x",
         "stylelint-no-unsupported-browser-features": "8.x",
         "stylelint-order": "6.x",
         "stylelint-scss": "6.x"
-    },
-    "devDependencies": {
-        "@stylistic/stylelint-config": "^2.0.0"
```
Comment (hisuwh) [inline packages/aquila-stylelint/package.json:29]: Surely it still needs to be a dev dependency?  It won't install peer dependencies?

Comment (hisuwh) [reply packages/aquila-stylelint/package.json]: Anything that this package requires to operate should be specified as a peer dependency.  I.e. anything we're specifying here https://github.com/AquilaLearning/aquila-linting/blob/main/packages/aquila-stylelint/README.md?plain=1#L14

Which is a point actually we need to add stylistic to this.

My point was more that in order to test this config within this repo we also need to install these, so they should also be devDependencies.

Comment (woahitsjc) [reply packages/aquila-stylelint/package.json:29]: As you mentioned, seems like we can install the peer dependency:

>UPDATE:
npm versions 1, 2, and 7 will automatically install peerDependencies if they are not explicitly depended upon higher in the dependency tree. For npm versions 3 through 6, you will receive a warning that the peerDependency is not installed instead.

https://nodejs.org/en/blog/npm/peer-dependencies

**Review by hisuwh — APPROVED** [review]

**Pull request merged**